easygcm-sample/src/main/java/eu/inloop/easygcm/sample/App.java

```diff
@@ -4,22 +4,20 @@
 import android.os.Bundle;
 
 import eu.inloop.easygcm.GcmListener;
-import eu.inloop.easygcm.WakeLockRelease;
 
 public class App extends Application implements GcmListener {
 
     @Override
-    public void onMessage(String s, Bundle bundle, WakeLockRelease wakeLockRelease) {
-        System.out.println("### message: " + s);
+    public void onMessage(String from, Bundle data) {
```

Comment:
So I spoke with @destil on this issue and we actually don't have to release the wakelock, it's done by the super class. The important thing is, that each onMessage is executed on a separate thread (spawned by AsyncTask.THREAD_POOL_EXECUTOR). 
We have to update the javadocs and documentation to make sure that the developers will know that onMessage is an asynchronous method.